### ISSUE TITLE
[Recipe] Add draining create:experience_nugget in create:item_drain to create kibe:liquid_xp

### ISSUE THREAD

**Issue by broman** [TeamAOF/All-of-Fabric-7#215]
### Describe your suggestion

I seem to recall in AOF6 you could drain the xp nuggets from Create into liquid XP, this was great because it let me run ores through the crushing wheel and the XP would be added to my "pool" of stored XP

### Why would you like this added/changed?

QOL

**Comment by Kazuhiko-Gushiken** [comment]
@broman there's a new mod called "Create: Enchantment Industry" which adds a "disenchanter" and if I remember correctly, it allows you to "drain" the xp nuggets in it.

**Comment by broman** [comment]
Yes, but that liquid xp is different from Kibe’s. So I have to use a Create pipe and a Kibe vacuum hopper as a hacky way to convert the fluid. It would be a nice to have.

**Comment by Kazuhiko-Gushiken** [comment]
really? for me i thought it gave kibe:xp i'll check later
